projects/demo/src/modules/experimental/button-vertical/examples/2/index.less

```diff
@@ -0,0 +1,26 @@
+@import '~@taiga-ui/core/styles/taiga-ui-local';
+
+button {
+    margin-bottom: 0.5rem;
+    width: 6.5rem;
+}
+
+[tuiAppearance][data-appearance='custom'] {
+    color: var(--tui-error-fill);
+    background: var(--tui-error-bg);
+
+    .appearance-hover({
+        background: var(--tui-error-bg-hover);
+    });
+
+    .appearance-active({
+        background: var(--tui-error-bg-hover);
+        opacity: 0.8;
+    });
+}
+
+.text {
+    width: 5rem;
+    height: 2rem;
+    white-space: pre-wrap;
```

Comment:
Should we bundle this into vertical button itself? I wonder if we can set it to 2 lines automatically somehow so people will just need to add fade.

Reply:
ok, if icon will be in :before, I can try to set max-height 2 * line-height to content